src/common/links.ts

```diff
@@ -58,6 +58,12 @@ export function isValidHttpUrl(link: string): boolean {
   }
 }
 
+export const domainOnly = (url: string): string => {
+  const { origin } = new URL(url);
+
+  return origin.replace(/(^\w+:|^)\/\//, '');
+};
```

Comment:
```suggestion
export const domainOnly = (url: string): string => {
  const { host } = new URL(url);

  return host;
};
```

Should work, no?

Reply:
Nvm...

Reply:
Wait, yeah, it'll work. I read the regex wrong, thought it removed `www.`

Reply:
`hostname` because `host` includes a port

Reply:
Yeah, changed to hostname 🙈

Reply:
Yeah hostname will do indeed :)